### PR TITLE
SST command when, the sst required frequence 0

### PR DESCRIPTION
**Pull request by Njord201**
I've check the value of sst command, if it's 0 i return sbp command.

### CODE REVIEW

**Review by Thomaltarix — DISMISSED** [review]

File: server/src/gui/communication/sst.c [hunk 1]
```diff
@@ -27,10 +27,11 @@ void sst_response(gui_t *gui, app_t *app, char *line)
     }
     line += LEN_COMMAND_AND_SPACE;
     freq = parse_positive_int_arg(line);
-    if (freq == CODE_ERROR_INVALID_NUMBER) {
+    if (freq == CODE_ERROR_INVALID_NUMBER || freq == 0) {
         sbp_command(gui);
         return;
     }
+    printf("sst %d\n", freq);
```
Comment (mathieurobert1) [inline server/src/gui/communication/sst.c:34]: Delete this debug line.

**Review by mathieurobert1 — CHANGES_REQUESTED** [review]

**Review by Thomaltarix — APPROVED** [review]

**Review by mathieurobert1 — APPROVED** [review]
A debug will be necessary but overall it's great.

**Comment by Njord201** [comment]
> A debug will be necessary but overall it's great.

FYM

File: server/src/gui/communication/sst.c (hunk 1 above)
Comment (mathieurobert1) [reply server/src/gui/communication/sst.c]: Are you serious ?

**Review by AxelF44 — APPROVED** [review]

**Pull request merged**